### PR TITLE
build: Fix some typos (most of them found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: build/integration/features/bootstrap/CommandLineContext.php [hunk 1]
```diff
@@ -95,7 +95,7 @@ private function findLastTransferFolderForUser($sourceUser, $targetUser) {
 	}
 
 	/**
-	 * @When /^transfering ownership from "([^"]+)" to "([^"]+)"/
+	 * @When /^transferring ownership from "([^"]+)" to "([^"]+)"/
 	 */
 	public function transferingOwnership($user1, $user2) {
```
Comment (stweil) [inline build/integration/features/bootstrap/CommandLineContext.php:100]: The function name still contains the typo. Where is this function used?

File: build/integration/features/bootstrap/CommandLineContext.php [hunk 2]
```diff
@@ -107,7 +107,7 @@ public function transferingOwnership($user1, $user2) {
 	}
 
 	/**
-	 * @When /^transfering ownership of path "([^"]+)" from "([^"]+)" to "([^"]+)"/
+	 * @When /^transferring ownership of path "([^"]+)" from "([^"]+)" to "([^"]+)"/
 	 */
 	public function transferingOwnershipPath($path, $user1, $user2) {
```
Comment (stweil) [inline build/integration/features/bootstrap/CommandLineContext.php:112]: The function name still contains the typo. Where is this function used?

File: build/integration/features/bootstrap/CommandLineContext.php (hunk 1 above)
Comment (kesselb) [reply build/integration/features/bootstrap/CommandLineContext.php:100]: `@When` is an annotation to map a step to a function.

`When transferring ownership from "user0" to "user1"` will call the method `transferingOwnershipPath` with $path, $user1 and $user2. 

Should be safe to rename the function as long as you keep the `@When` annotation.

Comment (stweil) [reply build/integration/features/bootstrap/CommandLineContext.php]: Thanks. I added a commit to fix the two function names, too.

**Review by danxuliu — APPROVED** [review]
:+1: 

Test failures are unrelated.

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**